src/auth/basic-auth.guard.ts

```diff
@@ -14,3 +14,14 @@ export class BasicAuthGuard implements CanActivate {
     );
   }
 }
+
+@Injectable()
+export class AdminWebhookGuard implements CanActivate {
```

Comment:
Maybe you can add some docs explaining the purpose of this guard

Reply:
Done https://github.com/safe-global/safe-events-service/pull/398/commits/3e94a4b0eb8a3f3af59680797d4f450efc0c6ed1